main.go

```diff
@@ -175,24 +190,18 @@ func main() {
 			log.Printf("Dumped system information to: %s", basePath)
 			return
 		}
-		cmd = exec.Command("rm", "-rf", basePath)
-		cmd.Run()
+
+		// The archive was created successfully, remove basePath. The
+		// error from os.RemoveAll is intentionally ignored, since there
+		// is no useful action we can do, and we don't need to confuse
+		// the user with an error message.
+		os.RemoveAll(basePath)
 
 		log.Printf("Dumped system information to: %s", basePath+".tar.gz")
 	}()
 
 	log.Println("Running tasks...")
 
-	// Avoid the creating goroutines and other controls if we're executing
-	// tasks sequentially.
-	if *maxParallelTasks == 1 {
```

Comment:
Am I right in thinking this was removed as if maxParallelTasks was set to 1, it will do it sequentially in the wait group anyway?